tests/testthat/helper-scope.R

```diff
@@ -3,29 +3,24 @@ renv_tests_scope <- function(packages = character(),
                              project = NULL,
                              envir = parent.frame())
 {
-  # source setup.R if necessary (for interactive scenarios)
-  running <- getOption("renv.tests.running", default = FALSE)
-  if (!running) {
-    path <- test_path("setup.R")
-    sys.source(path, envir = globalenv())
-  }
-
-  # use local repositories in this scope
+  # use private repositories in this scope
   renv_tests_scope_repos(envir = envir)
 
+  # use sandbox in this scope
+  # TODO(Kevin): Without this, tests fail trying to install BiocManager?
+  renv_scope_sandbox(envir = envir)
```

Comment:
It's worth thinking about whether it's better to apply that here, or in individual Bioc tests that need it.

Reply:
Good question... it might be better to turn this off, since this would better reflect real-life usage of `renv` as the sandbox won't get activated until a project is activated.

The BiocManager issues could also reflect real problems users might encounter.